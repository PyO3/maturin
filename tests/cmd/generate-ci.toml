bin.name = "maturin"
args = "generate-ci --help"
