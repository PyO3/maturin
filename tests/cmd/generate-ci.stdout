Generate CI configuration

Usage: maturin[EXE] generate-ci [OPTIONS] <CI>

Arguments:
  <CI>
          CI provider

          Possible values:
          - github: GitHub

Options:
  -m, --manifest-path <PATH>
          Path to Cargo.toml

  -v, --verbose...
          Use verbose output.
          
          * Default: Show build information and `cargo build` output. * `-v`: Use `cargo build -v`.
          * `-vv`: Show debug logging and use `cargo build -vv`. * `-vvv`: Show trace logging.
          
          You can configure fine-grained logging using the `RUST_LOG` environment variable.
          (<https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives>)

  -o, --output <PATH>
          Output path
          
          [default: -]

      --platform <platform>...
          Platform support
          
          [default: linux musllinux windows macos]

          Possible values:
          - all:        All
          - manylinux:  Manylinux
          - musllinux:  Musllinux
          - windows:    Windows
          - macos:      macOS
          - emscripten: Emscripten

      --pytest
          Enable pytest

      --zig
          Use zig to do cross compilation

  -h, --help
          Print help (see a summary with '-h')
